artview/components/nav.py

```diff
@@ -258,40 +266,91 @@ def AdvanceFileSelect(self, findex):
             return
         self.fileindex = findex
         self.filename = self.Vfilelist.value[findex]
-        self.menu._openfile(filename=self.filename)
```

Comment:
If we have redundant functionalist, we will have redundant code (I added a copy of this function in the end of the file). We shall not use code of an other component, and it is also better not, since there are normally some small differences.